.changeset/calm-wasps-accept.md

```diff
@@ -0,0 +1,22 @@
+---
+'@clerk/clerk-expo': minor
+---
+
+Introduce `createClerkClient` to avoid importing the Clerk class from clerk-js manually.
+
+This enables developers to create and access a Clerk instance in their application outside of React.
+```tsx
+
+import { ClerkProvider, createClerkClient } from "@clerk/expo"
+
+const clerkInstance = createClerkClient({ publishableKey: 'xxxx' })
+  
+// Be sure to pass the new instance to ClerkProvider to avoid running multiple instances of Clerk in your application
+<ClerkProvider Clerk={clerkInstance}>
```

Comment:
so going forward in Expo apps, every developer should create this new Clerk instance and pass it in _if they want to be able to access it outside of React_ right?

Reply:
Yeah that's right they should create the new instance with `createClerkClient`